README.md

```diff
@@ -0,0 +1,34 @@
+# PG Stream Copy
+[![Build Status](https://travis-ci.com/rtbhouse-apps/pg-stream-copy.svg?branch=master)](https://travis-ci.com/rtbhouse-apps/pg-stream-copy) [![codecov](https://codecov.io/gh/rtbhouse-apps/pg-stream-copy/branch/master/graph/badge.svg)](https://codecov.io/gh/rtbhouse-apps/pg-stream-copy)
+
+Fast COPY TO postgresql table directly from python by converting input data to bytes and stream to psycopg2 cursor using `COPY <table> FROM STDIN BINARY`
+
+### Usage:
+```python
+from datetime import date
+from psycopg2 import connect
+from pg_stream_copy import Schema, WriterEncoder
+
+
+conn = connect('postgresql://postgres@localhost')
+cursor = conn.cursor()
+table_name = 'public.example_table'
```

Comment:
Dodałbym jeszcze jakiegoś przykładowego SQL'a który tworzy tę tabelkę, żeby było widać skąd biorą się te linie niżej

Reply:
I added link to e2e tests in README: https://github.com/rtbhouse-apps/pg-stream-copy/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R34 should be sufficient

Reply:
I still think this would be better to show compact and complete usage scenario.